python-checks/src/test/resources/checks/inconsistentTypeHintImported.py

```diff
@@ -0,0 +1,5 @@
+class A:
+  v : int = 10
+
+class B:
+    def foo(self): ...
```

Comment:
_minor_ To be more explicit about the intention of the test, I suggest to rename the classes
```suggestion
class ClassWithFieldOnly:
  v : int = 10

class ClassWithMethodOnly:
    def foo(self): ...
```